__tests__/lib/getPkgVersion.test.js

```diff
@@ -0,0 +1,28 @@
+const { getPkgVersion } = require('../../lib/getPkgVersion');
+
+jest.mock('../../package.json', () => ({ version: 1.1 }), {
+  virtual: true, // needed to mock a JSON file
+});
+
+describe('#getPkgVersion()', () => {
+  beforeEach(() => {
+    jest.resetModules();
+  });
+
+  // source adapted from `rdme`:
+  // https://github.com/readmeio/rdme/blob/HEAD/__tests__/lib/getPkgVersion.test.ts
+  describe('from actual package.json', () => {
+    // eslint-disable-next-line global-require
+    const pkg = require('../../package.json');
+
+    it('should grab version', () => {
+      return expect(getPkgVersion()).toBe(String(pkg.version));
+    });
+  });
+
+  describe('from mocked package.json', () => {
+    it('should parse numeric version from package.json as string', () => {
```

Comment:
Wouldn't it always be a string in the package.json? 

<img width="1136" alt="image" src="https://github.com/readmeio/readme-micro/assets/848223/c40e018d-e349-4ba8-9229-d86f0aca6e56">

Running `npm init -y` puts a string in there, and my editor warns when it isn't a string. I dont think i've ever seen a non-string value in a package.json either.

Reply:
good point, will remove test. editors will warn if it's a X.Y or X.Y.Z. I've seen just plain integers used for X:

```json
  "version": 2,
```

agreed now - it's overkill. the other tests are sufficient.

Reply:
and, doy, this is for micro's package - silly me